std/algorithm/mutation.d

```diff
@@ -2012,6 +2078,20 @@ if (s == SwapStrategy.stable
     }
 }
 
+@safe unittest
+{
+    char[] chars = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'];
+    remove(chars, 4);
+    assert(chars == ['a', 'b', 'c', 'd', 'f', 'g', 'h', 'h']);
+
+    char[] bigChars = "∑œ∆¬é˚˙ƒé∂ß¡¡".dup;
+    assert(remove(bigChars, tuple(4, 6), 8) == ("∑œ∆¬˙ƒ∂ß¡¡"));
+
+    import std.exception : assertThrown;
+    assertThrown(remove(bigChars.dup, 1, 0));
```

Comment:
FWIW there's a PR to deprecate this because it's really error-prone: https://github.com/dlang/phobos/pull/6154/files

Reply:
Ok, makes sense. Whichever one gets merged first will have to adapt to the other.